src/main/java/jnr/posix/MacOSPOSIX.java

```diff
@@ -30,7 +36,12 @@ public long sysconf(Sysconf name) {
     public Times times() {
         return NativeTimes.times(this);
     }
-    
+
+    @Override
+    public Pointer environ() {
+        return environ._NSGetEnviron().getPointer(0);
```

Comment:
Is this a `char ***`?  I'm a  bit confused why you need to dereference the first pointer.